Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -704,6 +704,8 @@ def identitySim(wf):
 # INPUT command for reminiAOD wfs on UL-like relval inputs
 steps['TTbar_13_reminiaod2016UL_preVFP_INPUT']={'INPUT':InputInfo(dataSet='/RelValTTbar_13UP16/CMSSW_10_6_12-PU25ns_106X_mcRun2_asymptotic_preVFP_v8_hltul16_preVFP-v1/AODSIM',label='rmaod',location='STD')}
 steps['TTbar_13_reminiaod2016UL_postVFP_INPUT']={'INPUT':InputInfo(dataSet='/RelValTTbar_13UP16/CMSSW_10_6_12-PU25ns_106X_mcRun2_asymptotic_v13_hltul16_postVFP-v1/AODSIM',label='rmaod',location='STD')}
+# INPUT command for reminiAOD wfs on PbPb relval inputs 
+steps['HydjetQ_reminiaodPbPb2021_INPUT']={'INPUT':InputInfo(dataSet='/RelValHydjetQ_B12_5020GeV_2021_ppReco/CMSSW_11_2_0_pre1-PU_111X_mcRun3_2021_realistic_HI_v5-v1/GEN-SIM-RECO',label='rmaod',location='STD')}
```

Comment:
My goal from #30349 was to provide a reminiAOD setup using the target input dataset, which in this case would be from 10_3_X.
This one is apparently from 11_2_0, which makes this a bit impractical for the needs of reminiAOD

Reply:
Ok, I assumed we would just do then when backporting to 10_6_X.  But ok, we can run on a 10_3 input in the master.  Will just have to fix the puppi issue.

Reply:
I can input instead the dataset: /RelValHydjetQ_B12_5020GeV_2018_ppReco/CMSSW_10_3_3-PU_103X_upgrade2018_realistic_HI_v11-v1/GEN-SIM-RECO
However, it's only at T1_US_FNAL_Disk.
What does one enter in the function InputInfo, for the argument 'location' instead of 'STD'?
@chayanit  maybe you can give me a pointer here?

Reply:
is there an AOD file instead, to stay as close as possible to the (re)miniAOD scope?

Reply:
No, it seems we are still writing RECOSIM in all the relval workflows.